app/src/main/kotlin/io/homeassistant/companion/android/util/InsetsUtil.kt

```diff
@@ -20,6 +21,26 @@ fun WindowInsets.bottomPaddingValues(): PaddingValues {
     return only(WindowInsetsSides.Bottom).asPaddingValues()
 }
 
+@Composable
+fun safeBottomWindowInsets(): WindowInsets {
+    return WindowInsets.safeDrawing.only(WindowInsetsSides.Bottom + WindowInsetsSides.Horizontal)
+}
+
+@Composable
+fun safeBottomPaddingValues(): PaddingValues {
+    return safeBottomWindowInsets().asPaddingValues()
+}
+
+@Composable
+fun safeTopWindowInsets(): WindowInsets {
+    return WindowInsets.safeDrawing.only(WindowInsetsSides.Top + WindowInsetsSides.Horizontal)
+}
+
+@Composable
+fun safeTopPaddingValues(): PaddingValues {
+    return safeTopWindowInsets().asPaddingValues()
+}
```

Comment:
@jpelgrom I think I should double check in settings screen the insets because I didn't handle the case of cutout corner like you've said in barcode PR.